.github/workflows/ci.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           CI: true
 
       - name: Sonar Code Analysis
-        if: "github.repository == 'pioardi/poolifier'"
+        if: "github.repository == 'pioardi/poolifier' && matrix.node-version == '12.x'"
```

Comment:
I would prefer to have it run on the current node.js LTS, not the previous one.

Reply:
We are also releasing with node 12

https://github.com/pioardi/poolifier/blob/5716dd56de3de46c5871a6af3b9f02da897bc2e4/.github/workflows/npmpublish.yml#L47

@pioardi What is your opinion about that?

I'm not biased at all and don't care which version is used in the end :slightly_smiling_face:

Reply:
I would prefer to support node 14.x.x better :)  In this case I don't think that really matter a lot, but please use the 14 version.

Reply:
```suggestion
        if: "github.repository == 'pioardi/poolifier' && matrix.node-version == '14.x'"
```